reactivesocket-cpp/test/simple/CancelSubscription.h

```diff
@@ -0,0 +1,22 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#pragma once
+
+
+#include <reactivesocket-cpp/src/ReactiveStreamsCompat.h>
+#include <reactivesocket-cpp/src/mixins/IntrusiveDeleter.h>
+
+namespace reactivesocket {
+    class CancelSubscription
```

Comment:
The name CancelSubscription suggests that there will be something cancelled. But looking at the implementation, it is an empty implementation. Can you consider renaming this to EmptySubscription or NullSubscription?